cmd/res/profiles/camera.yaml

```
@@ -162,6 +162,17 @@ deviceResources:
     properties:
       valueType: "Object"
       readWrite: "W"
+  - name: "CustomMetadata"
+    isHidden: false
+    description: "Get and set custom added information about the cameras"
+    attributes:
+      service: "Device"
+      custom: "Custom"
```

I recommend removing the custom field, and setting service to `EdgeX`. this will align better with the other custom funcs
```suggestion
      service: "EdgeX"
```

---

Yes, please refer to the doc and add the `GetCustomMetadata` and `SetCustomMetadata` to the README file.
https://github.com/edgexfoundry/device-onvif-camera/blob/0a79c5a718067479237195976121108915350925/README.md?plain=1#L97